### ISSUE TITLE
Selected or hovered items are not visible when using custom colours

### ISSUE THREAD

**Issue by edwardhorsford** [alphagov/accessible-autocomplete#203]
If a user uses custom colours, the selected / hovered item is not visible - is there anything we can do about this? Invisible outline around selected item like we do with buttons?

Example where United Kingdom is selected:
![screen shot 2017-07-21 at 14 26 05](https://user-images.githubusercontent.com/2204224/28465444-b7781f70-6e20-11e7-974f-0ca99413721d.png)

NB: if using arrow keys and hover, it's possible to get in to a state where there's a visible dotted line - don't know if this is related or not. Example:
![screen shot 2017-07-21 at 14 28 01](https://user-images.githubusercontent.com/2204224/28465505-e47fc536-6e20-11e7-8f59-78d10f513bf7.png)

**Comment by tvararu** [comment]
I completely forgot about this, we found it on our trip to DAC too. Good find.

Yes, there should be solutions for this.

**Comment by vanitabarrett** [comment]
I think this issue is already covered by https://github.com/alphagov/accessible-autocomplete/issues/483 and https://github.com/alphagov/accessible-autocomplete/issues/369, so I'm going to close this for now. Please re-open this ticket if you feel the issue is separate.
